modules/admin_manual/pages/maintenance/backup_and_restore/backup.adoc

```diff
@@ -1,69 +1,94 @@
-. the `data/` directory containing web-server-related files, `owncloud.log` and possibly encryption keys,
-. a fresh ownCloud database dump,
-. and if applicable custom theme files which are usually located in the `apps` or `apps-external` directory (see xref:developer_manual:core/theming.adoc[Theming ownCloud]).
+.  The `config/` directory.
+.  The `data/` directory. +
+Note that the `data/` directory may not only contain user files, but also keys for encryption.
+.  The `apps/` directory. +
+Note that this is only necessary if you are not using the `apps-external/` directory and have added own apps or themes.
+.  The `apps-external/` directory. +
+Note that this is only necessary if it exists and is in use.
+.  The ownCloud database.
+.  The custom theme files, if you had any. See xref:developer_manual:core/theming.adoc[Theming ownCloud]. +
+Note that theme files are usually located in either the `apps/` or `apps-external/` directory.
 
-If you install your ownCloud server from our
-https://download.owncloud.org/download/repositories/stable/owncloud/[Open
-Build Service] packages (or from distro packages, which we do not
-recommend), *do not backup your ownCloud server files*, which are the
-other files in your `owncloud/` directory such as `core/`, `3rdparty/`,
-`apps/`, `lib/`, and all the rest of the ownCloud files. If you restore
-these files from backup they may not be in sync with the current package
-versions and will fail the code integrity check. This may also cause
-other errors.
+IMPORTANT: If you have customized user home direcories or a custom location for encryption keys, you have to manually take care of backing them up and restoring them to the same location.
 
-If you install ownCloud from the source tarballs, this will not be an
-issue, and you can safely backup your entire ownCloud installation, with
-the exception of your ownCloud database. Databases cannot be copied, instead
-you must use the database tools to make a correct database dump.
+== Prerequisites
 
-To restore your ownCloud installation from backup, see xref:maintenance/backup_and_restore/restore.adoc[Restoring ownCloud].
+To ensure a consistent backup, stop your webserver to prevent users trying to access ownCloud via the web. As an alternative, you can stop serving the virtual host for ownCloud:
 
-== Backing up the config/ and data/ Directories
+[source,console,subs="attributes+"]
+----
+sudo service apache2 stop
+----
+
+== Backup Scenarios
+
+Tarball Installation::
+. If you have installed ownCloud from a tarball, you can safely backup the entire installation, with the exception of your ownCloud database. Databases cannot be copied, instead you must use the database tools to make a correct backup.
+
+. You may also backup only the directories named above and the database. To avoid issues, you have to use the same tarball version as the ownCloud version when restoring.
```

Comment:
```suggestion
. You can also back up only the directories mentioned above and the database. To avoid issues, you have to use the same tarball version as the ownCloud version when restoring.
```